### PR TITLE
converted all f-strings to % for logging an mentioned in issue #1205

### PR DESCRIPTION
**Pull request by DipanshuShukla**
converted all f-strings to % for logging an mentioned in issue #1205

### CODE REVIEW

**Comment by DipanshuShukla** [comment]
why did these tests fail?

**Comment by ReimarBauer** [comment]
Have a look on the "Details" for flake8
https://github.com/Open-MSS/MSS/runs/6059325533?check_suite_focus=true
The other two I Iook at.

File: mslib/mscolab/file_manager.py [hunk 1]
```diff
@@ -151,7 +151,7 @@ def update_operation(self, op_id, attribute, value, user):
         operation = Operation.query.filter_by(id=op_id).first()
         if attribute == "path":
             if value.find("/") != -1 or value.find("\\") != -1 or (" " in value):
-                logging.debug("malicious request: %s", user)
+                logging.debug("malicious request: %s" % user)
```
Comment (ReimarBauer) [inline mslib/mscolab/file_manager.py]: the "," should be correct here

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
Have a look on the oldstyle syntax and check also erros of tests

**Comment by ReimarBauer** [comment]
> why did these tests fail?

the dockerimage was not correct, fixed that temporary (PR is open). But tests have errors, please 
check.
You can also run tests locally, see
https://mss.readthedocs.io/en/stable/development.html?highlight=pytest#run-tests

**Comment by ReimarBauer** [comment]
@DipanshuShukla please have a look on https://docs.python.org/3/howto/logging.html#logging-variable-data

**Comment by ReimarBauer** [comment]
we can reopen it when continuing

**Pull request closed without merging**